OpenRA.Game/Settings.cs

```diff
@@ -224,6 +224,10 @@ public class KeySettings
 		public Hotkey ScatterKey = new Hotkey(Keycode.X, Modifiers.Ctrl);
 		public Hotkey DeployKey = new Hotkey(Keycode.F, Modifiers.None);
 		public Hotkey StanceCycleKey = new Hotkey(Keycode.Z, Modifiers.Ctrl);
+		public Hotkey StanceHoldFireKey = new Hotkey(Keycode.UNKNOWN, Modifiers.None);
```

Comment:
The later C&C games mapped `Alt + A`, `Alt + S`, `Alt  + D`, `Alt + F` respectively for AttackAnything, Defend, ReturnFire, HoldFire.  We should use the same defaults.